zerver/webhooks/github/view.py

```diff
@@ -550,6 +550,10 @@ def get_event(request: HttpRequest, payload: Dict[str, Any], branches: Optional[
     elif event in IGNORED_EVENTS:
         return None
 
+    # More versbose error reporting
+    if event == "pull_request":  # nocoverage
+        event = "{}:{}".format(event, payload["action"])
```

Comment:
Maybe this commit should just add `payload['action'']` if present  for all event types?  I don't think `pull_request` is the only category with an `action` field.